src/form/v2/birth/forms/print-certificate.ts

```diff
@@ -531,13 +534,128 @@ export const BIRTH_CERTIFICATE_COLLECTOR_FORM = defineActionForm({
       },
       fields: [
         {
-          id: 'collector.collect.payment.data',
+          id: 'collector.collect.payment.data.after365',
           type: FieldType.DATA,
           label: {
             defaultMessage: 'Payment details',
             description: 'Title for the data section',
             id: 'v2.event.birth.action.certificate.form.section.collectPayment.data.label'
           },
+          conditionals: [
+            {
+              type: ConditionalType.SHOW,
+              conditional: and(
+                not(
+                  field('child.dob')
+                    .isAfter()
+                    .days(applicationConfig.BIRTH.LATE_REGISTRATION_TARGET)
+                    .inPast()
+                ),
+                field('child.dob').isBefore().now()
```

Comment:
Why is the `field('child.dob').isBefore().now()` needed in the conditionals?

Reply:
The date of birth is before today (not in the future)